domains/sharans.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "sharansharma94",
+        "email": "sharansharma19@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://sharansharma94.github.io/portfolio/"
```

CNAME can't contain protocol or `/`
```suggestion
        "CNAME": "sharansharma94.github.io"
```